src/frontend/app/services/FileService.js

```diff
@@ -79,18 +79,33 @@ export default class FileService {
             let beautiful = this.toBeautifulSyntax.toBeautifulSyntax(data);
             data = beautiful;
 
-            cur.codeController.setEditorValue(beautiful);
-
-
             let adapterName = data.match(/<Adapter[^]*?name=".*?"/g);
-            adapterName = adapterName[0].match(/".*?"/g)[0].replace(/"/g, '');
 
-            localStorage.setItem('currentAdapter', adapterName);
+            if (adapterName != null) {
+                adapterName = adapterName[0].match(/".*?"/g)[0].replace(/"/g, '');
+                localStorage.setItem('currentAdapter', adapterName);
+            }
 
+            cur.codeController.setEditorValue(beautiful);
             cur.codeController.quickGenerate();
         }).catch(e => {
-            console.error('Error getting single file: ', e);
+            console.error('Error getting file: ', e);
+        })
+    }
+
+    getFile(deployableUnit, name) {
```

Comment:
Er zijn twee methods om een file op te halen, deze is simpeler en haalt alleen maar puur de data op.